Libraries/Storage/AsyncStorage.js

```diff
@@ -105,6 +105,22 @@ var AsyncStorage = {
   /**
    * Merges existing value with input value, assuming they are stringified json.
    * Returns a `Promise` object. Not supported by all native implementations.
+   *
+   * Example:
+   * ```JavaScript
+   *   let k1 = 'key_1'
+   *   let val_1a = JSON.stringify({'a': '123', 'b': {'b1': '234', 'b2': '234'}})
+   *   let val_1b = JSON.stringify({'b': {'b2': '555', 'b3': '555'}})
+   *   let EXPECTED_RESULT = {'a':'123','b':{'b1':'234','b2':'555','b3':'555'}}
```

Comment:
Instead of a/b and 123/234/555 could you use some more realistic names and values. Otherwise it is pretty hard to figure out what is going on. I have to spend a lot of time read through the two json dumps to figure things out.